docs/get-started/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: layouts/pages/basic.njk
+title: Overview
+order: 0
+tags:
+  - getstarted
+---
+
+<style>
+  #get-started-nav {
+    display: grid;
+    grid-template-columns: 1fr;
+    gap: var(--rh-space-2xl, 32px);
+    margin-block-start: var(--rh-space-2xl, 32px);
+  }
+
+  #get-started-nav figcaption {
+    font-family: var(--rh-font-family-heading, RedHatDisplay, 'Red Hat Display', 'Noto Sans Arabic', 'Noto Sans Hebrew', 'Noto Sans JP', 'Noto Sans KR', 'Noto Sans Malayalam', 'Noto Sans SC', 'Noto Sans TC', 'Noto Sans Thai', Helvetica, Arial, sans-serif);
+    font-size: var(--rh-font-size-heading-sm, 1.5rem);
+  }
+
+  @container container (min-width: 567px) {
+    #get-started-nav {
+      grid-template-columns: 1fr 1fr;
+    }
+  }
+
+</style>
+
+## Get started
+
+The Red Hat Design System for digital experiences gives designers and developers the tools to create accessible and on-brand user interfaces quickly. The role-specific pages linked below will guide you through using the design system.
+
+<nav id="get-started-nav">
```

Comment:
Since there can be more that one <nav> on a page, the need unique `aria-label` elements. I suggest `<nav id="get-started-nav" aria-label="Get started">`